src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -19,7 +20,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return FrameworkName;
+                return string.Format("{0} {1}", FrameworkName, typeof(object).GetTypeInfo().Assembly.GetName().Version);
```

Comment:
I don't have the absolute latest code on my Linux machine, when I run:

```C#
    static void Main(string[] args)
    {
        Console.WriteLine(typeof(object).GetTypeInfo().Assembly.GetName().Version);
    }
```

I get the output:

```
4.0.0.0
```

This seems less useful to me.